src/Dnt.Commands/ProjectCommandBase.cs

```diff
@@ -18,7 +18,7 @@ protected string[] GetProjectPaths()
                 if (Path.EndsWith(".sln"))
                 {
                     var solution = SolutionFile.Parse(System.IO.Path.GetFullPath(Path));
-                    return solution.ProjectsInOrder.Select(p => p.AbsolutePath).ToArray();
+                    return solution.ProjectsInOrder.Where(p => p.ProjectType != SolutionProjectType.SolutionFolder).Select(p => p.AbsolutePath).ToArray();
```

Comment:
I'm not 100% sure this is okay for the other commands? I was running into issues with my test sln where it was trying to load a solution folder as a project.

Reply:
I think this might even fix other commands - the idea is to only load "real" projects here...